docs/credential-modeling.md

```diff
@@ -57,7 +56,7 @@ To create a structure matching the W3C Verifiable Credential [JSON-LD Syntax](ht
 
 2. Add to the `.vc` object:
 
-   ```
+   ```js
```

Comment:
```suggestion
   ```
```

I don't think this block is a complete JSON structure (no opening braces) so this format won't quite work

Reply:
Ah good point, thanks.  Fixed up and re-pushed.